contracts/ReaperAutoCompoundScreamLeverage.sol

```diff
@@ -341,24 +341,19 @@ contract ReaperAutoCompoundScreamLeverage is ReaperBaseStrategy {
      *      It is not 100% accurate as it uses current balances in Compound to predict into the past.
      */
     function predictScreamAccrued() public view returns (uint256) {
-        // Has no previous log to compare harvest time to
-        if (harvestLog.length == 0) {
+        // Has no previous harvest to calculate accrual
+        if (lastHarvestTimestamp == 0) {
             return 0;
         }
+
         (uint256 supplied, uint256 borrowed) = getCurrentPosition();
         if (supplied == 0) {
             return 0; // should be impossible to have 0 balance and positive comp accrued
         }
 
         uint256 distributionPerBlock = comptroller.compSpeeds(address(cWant));
-
         uint256 totalBorrow = cWant.totalBorrows();
-
-        //total supply needs to be exchanged to underlying using exchange rate
-        uint256 totalSupplyCtoken = cWant.totalSupply();
-        uint256 totalSupply = totalSupplyCtoken
-            * cWant.exchangeRateStored()
-            / MANTISSA;
+        uint256 totalSupply = totalBorrow + cWant.getCash();
```

Comment:
As per the formula we used in the deleveraging bot to calculate `totalSupply`

Reply:
I prefer not to make changes that just improve code readability in a production system with a lot of funds at risk.